github_pr_magic_nb/MagicChecker.py

```diff
@@ -1,11 +1,12 @@
+import re
+
 def is_magic_value(value):
     if value < 100:
         # Do not process non-active repositories
         return False
-    divider = 1000 if value < 100000 else 10000
-    if value % divider == 0:
-        return True
     value_str = str(value)
+    if re.fullmatch("\d0+", str(value)):
```

Comment:
We would like to exclude multiple of 1k when the number of PR is growing too fast.
Additionally, the string of value was already declared :)

Reply:
This solution is designed to scale with the project. It will only take into account PRs that have number on the order of magnitude of the speed of development. I will write up an RFC later on.